hajji/hello.S

```diff
@@ -61,6 +67,7 @@ start:
         callw clrscr                  # clear screen
         callw curshome                # move cursor home - top:left
         callw greeting                # display a greeting string
+		cli
```

Comment:
It wasn't a typo since I wrote the code to turn interrupts off only when necessary and enable them when appropriate to do so.